packages/ui/src/lib/button/Button.svelte

```diff
@@ -1,3 +1,49 @@
-<button class="btn bg-core-blue-600 p-4">
-    <slot></slot>
-</button>
+<script lang="ts">
+	import classNames from 'classnames';
+
+	export let style: 'primary' | 'secondary' | 'ghost' = 'primary';
+	export let size: 'xs' | 'sm' | 'base' | 'lg' | 'xl' = 'base';
+	export let href: string | undefined = undefined;
+	export let type: 'button' | 'submit' = 'button';
```

Comment:
When used as a link the button component doesn't currently work corretly, as the `href` attribute isn't set on the `<a>` element that gets created.

Also, I'm not sure that creating navigation links that are visually indistinguishable from buttons that trigger actions is actually a good idea - it may be quite confusing for users.

See also https://github.com/Greater-London-Authority/ldn-viz-tools/pull/48

Reply:
See 'Start exploring the data' CTA for (valid imho) use case: https://apps.london.gov.uk/survey-of-londoners/

Reply:
It's a fair use case.

We could consider differing styling for page navigation buttons. We could try an arrow icon to imply you're taken somewhere when clicked?